ci/infra/testrunner/skuba/skuba.py

```diff
@@ -187,4 +187,4 @@ def _run_skuba(self, cmd, cwd=None):
 
         env = {"SSH_AUTH_SOCK": os.path.join(self.conf.workspace, "ssh-agent-sock")}
 
-        return self.utils.runshellcommand(self.binpath + " "+ cmd, cwd=cwd, env=env)
+        return self.utils.runshellcommand(self.binpath + f" -v {verbosity} " + cmd, cwd=cwd, env=env)
```

Comment:
maybe make this whole thing an f string? f"{self.binpath} -v {verbosity} {cmd}"

Reply:
Doh.  I was so excited about changing as little as possible... :)

Reply:
Updated